content/patterns/checkbox/examples/checkbox-mixed.html

```diff
@@ -18,103 +18,104 @@
-            Instead of using transparency, the focused element has a thicker border and less padding.
-            When an element receives focus, its border changes from 0 to 2 pixels and padding is reduced by 2 pixels.
-            When an element loses focus, its border changes from 2 pixels to 0 and padding is increased by 2 pixels.
+            オペレーティングシステムのハイコントラスト設定が有効になっている場合、一部のシステムでは透明に設定された境界線が可視化されるため、フォーカスされている要素と他の要素との間に視覚的な違いを作成するために透明度を使用することはできません。
+            透明度を使用する代わりに、フォーカスされた要素はより太い境界線とより少ないパディングを持ちます。
+            要素がフォーカスを受け取ると、その境界線は0から2ピクセルに変わり、パディングは2ピクセル減少します。
+            要素がフォーカスを失うと、その境界線は2ピクセルから0に変わり、パディングは2ピクセル増加します。
           </li>
           <li>
-            To ensure the borders of the inline SVG checkbox graphics in the CSS have sufficient contrast with the background when high contrast settings invert colors, the color of the borders are synchronized with the color of the text content.
-            For example, the color of the checkbox borders is set to match the foreground color of high contrast mode text by specifying the CSS <code>currentcolor</code> value for the <code>stroke</code> property of the <code>rect</code> and <code>polyline</code> elements used to draw the checkbox.
-            To make the background of the checkbox graphics match the high contrast background color, the <code>fill-opacity</code> attribute of the <code>rect</code> element is set to zero.
-            If specific colors were instead used to specify the <code>stroke</code> and <code>fill</code> properties, those colors would remain the same in high contrast mode, which could lead to insufficient contrast between the checkbox and the background or even make the checkbox invisible if the color matched the high contrast mode background.
-            <br>Note: The SVG element needs to have the CSS <code>forced-color-adjust</code> property set to <code>auto</code> for the <code>currentcolor</code> value to be updated in high contrast mode.
-            Some browsers do not use <code>auto</code> for the default value.
+            ハイコントラスト設定で色が反転した場合に、CSS のインライン SVG チェックボックスグラフィックの境界線が背景と十分なコントラストを持つことを保証するために、境界線の色はテキストコンテンツの色と同期されます。
+            たとえば、チェックボックスの境界線の色は、チェックボックスを描画するために使用される <code>rect</code> 及び <code>polyline</code> 要素の <code>stroke</code> プロパティに CSS <code>currentcolor</code> 値を指定することにより、ハイコントラストモードテキストの前景色と一致するように設定されます。
+            チェックボックスグラフィックの背景をハイコントラストの背景色と一致させるには、<code>rect</code> 要素の <code>fill-opacity</code> 属性をゼロに設定します。
+            代わりに特定の色を使用して <code>stroke</code> 及び <code>fill</code> プロパティを指定した場合、それらの色はハイコントラストモードで同じままであり、チェックボックスと背景の間のコントラストが不十分になったり、色がハイコントラストモードの背景と一致した場合はチェックボックスが見えなくなったりする可能性があります。
+            <br>注：SVG 要素は、ハイコントラストモードで <code>currentcolor</code> 値を更新するために、CSS <code>forced-color-adjust</code> プロパティを <code>auto</code> に設定する必要があります。
```

Comment:
WCAG の翻訳では Note は「注記」で統一しているので、合わせてみました。
```suggestion
            <br>注記：SVG 要素は、ハイコントラストモードで <code>currentcolor</code> 値を更新するために、CSS <code>forced-color-adjust</code> プロパティを <code>auto</code> に設定する必要があります。
```